packages/typespec-test/test/ai/generated/typespec-ts/README.md

```diff
@@ -1,75 +1,43 @@
-# Azure Client client library for JavaScript
+# Azure Projects REST client library for JavaScript
 
-This package contains an isomorphic SDK (runs both in Node.js and in browsers) for Azure Client client.
 
 
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
```

Comment:
is this sentence also auto generated?
Can we change it to:  Please make sure to refer to our Rest client docs when using this library

Reply:
After discussing with QQ, the wording is given by Jose, let's use a sperated issue for the wordings review. https://github.com/Azure/autorest.typescript/pull/3037